.storybook/stories/useCameraShake.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import * as THREE from 'three'
+import { useFrame, Camera } from 'react-three-fiber'
+
+import { Setup } from '../Setup'
+
+import { useCameraShake, ShakeConfigPartial, PerspectiveCamera } from '../../src'
+
+export default {
+  title: 'Camera/useCameraShake',
+  component: useCameraShake,
+  decorators: [
+    (storyFn) => (
+      <Setup cameraPosition={new THREE.Vector3(0, 0, 10)} controls={false}>
+        {storyFn()}
+      </Setup>
+    ),
+  ],
+}
+
+function Scene() {
+  const cube = React.useRef<THREE.Mesh>()
+
+  useFrame(() => {
+    if (cube.current) {
+      cube.current.rotation.x = cube.current.rotation.y += 0.01
+    }
+  })
+
+  return (
+    <>
+      <mesh ref={cube}>
+        <boxBufferGeometry args={[2, 2, 2]} />
+        <meshStandardMaterial wireframe color="white" />
+      </mesh>
+      <mesh position={[0, -6, 0]} rotation={[Math.PI / -2, 0, 0]}>
+        <planeBufferGeometry args={[200, 200, 75, 75]} />
+        <meshBasicMaterial wireframe color="red" side={THREE.DoubleSide} />
+      </mesh>
+    </>
+  )
+}
+
+function UseCameraShakeDemo({ cfg }) {
+  const baseCamRef = React.useRef<Camera>()
+  useCameraShake(baseCamRef.current, cfg)
+
+  return (
+    <>
+      <PerspectiveCamera makeDefault ref={baseCamRef} position={[0, 0, 20]} />
+      <React.Suspense fallback={null}>
+        <Scene />
+      </React.Suspense>
+    </>
```

Comment:
```suggestion
  const { camera } = useThree()
  useCameraShake(camera, cfg)

  return (
    <>
      <OrbitControls />
      <React.Suspense fallback={null}>
        <Scene />
      </React.Suspense>
    </>
```

1. show how the default camera can be used
2. add OrbitControls to show that it doesn't conflict

Not sure about point 1, honestly, let me know what you think :D